Simplified/NYPLAccountSignInViewController.m

```diff
@@ -831,7 +831,7 @@ - (void)validateCredentials
   NSURLSessionDataTask *const task =
     [self.session
      dataTaskWithRequest:request
-     completionHandler:^(NSData *data,
+     completionHandler:^(NSData *data __attribute__((unused)),
```

Comment:
This notation is more concise, so let's use this instead:

```Objective-C
NSURLSessionDataTask *const task =
  [self.session
   dataTaskWithRequest:request
   completionHandler:^(__unused NSData *data,

Reply:
so @winniequinn you're saying to use the unused attribute instead of discardable?

Reply:
I'm just suggesting we use `__unused` instead of `__attribute__((unused))` as per the example.